ska_trend/fid_drop_mon/index_template.html

```diff
@@ -0,0 +1,123 @@
+<HTML>
+    <HEADER>
+        <html>
+            <head>
+                <meta name="viewport" content="width=device-width, initial-scale=1">
+                <meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
+                <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+                rel="stylesheet"
+                integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+                crossorigin="anonymous">
+
+                <style>
+                    h1 {
+                        color: #990000;
+                    }
+
+                    h2 {
+                        color: #990000;
+                    }
+                    .content-container {
+                    margin-left: 5px; /* Adjust the value as needed */
+                    }
+
+                </style>
+      <title>Fid Light Drops</title>
+
+    </HEADER>
+    <BODY>
+
+            <!--#include virtual="/incl/header.html"-->
+<div class="container-md content-container">
+<H3>Fid Light Drops</H3>
+
+<div class="accordion" id="accordionExample">
+    <div class="accordion-item">
+        <h4 class="accordion-header" id="heading-table">
+            <button class="accordion-button collapsed bg-primary bg-opacity-25" type="button" data-bs-toggle="collapse"
+                data-bs-target="#info-table" aria-expanded="true" aria-controls="info-table">
+                Table Information
+            </button>
+        </h4>
+        <div id="info-table" class="accordion-collapse collapse" aria-labelledby="info-table"
+            data-bs-parent="#accordionExample">
+            <div class="accordion-body bg-primary bg-opacity-10">
+                The table below is of fid drop events.
+                <UL>
+                    <LI>Interval Start : start of tracking interval - usually the start of Kalman for the obsid dwell,
+                        but if there is a NMAN transition in an interval this can be a later NPNT transition at the same pointing. </LI>
+                    <LI>Interval Stop : end of tracking interval - usually transition to NMAN</LI>
+                    <LI>Obsid : Obsid assigned to the maneuver event to this dwell by kadi events</LI>
+                    <LI>Fid Slot : Fid slot number</LI>
+                    <LI>Track Fraction : Fraction of the interval that the fid light was tracked (AOACFCTN == "TRAK")</LI>
+                    <LI>Notes : Any notes from the google sheet</LI>
+                </UL>
+
+                The notes are from the google sheet at <a href="{{sheet_url}}">{{sheet_url}}</a>.
```

Comment:
This currently links to the full sheet which shows the `acdc_annotated` table by default. Can you link to the one with the correct `gid` in the URL?